scripts/add-test-user.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 NETWORK=hint_nw
 
-image=mrcide/hint-user-cli:mrc-373_userrepo #TODO: change this to master after merge
+image=mrcide/hint-user-cli:mrc-372 #TODO: change to master when merged
```

Comment:
...hence changing this line to the current branch rather than master, as it doesn't exist on the master branch yet!